iam_openapi_spec.yml

```diff
@@ -4607,6 +4671,22 @@ components:
             Expiry in seconds from the time of generation. If not provided, the token will have the expiry of 24 hours
           type: integer
 
+    SsoLoginRequest:
+      type: object
+      additionalProperties: false
+      required:
+        - domain
+        - redirectUri
+      properties:
+        domain:
+          description: Domain of the user's organization
+          type: string
+          maxLength: 320
+        redirectUri:
+          description: callback url for generating authentication url
+          type: string
+          maxLength: 320
```

Comment:
Why 320? Is this workos specified limit?

Reply:
added for safe check. Removed now.